scripts/mcp/test_rag.sh

```
@@ -0,0 +1,215 @@
+    fi
+}
+
+log_verbose() {
+    if [ "$VERBOSE" = true ]; then
+        echo "$@"
+    fi
+}
+
+log_success() {
+    if [ "$QUIET" = false ]; then
+        echo -e "${GREEN}✓${NC} $@"
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)